packages/esm-ward-app/src/ward-view/ward-view.component.tsx

```diff
@@ -15,11 +14,13 @@ const WardView = () => {
   const { sessionLocation } = useSession();
   const allLocations = useLocations();
   const { t } = useTranslation();
+  const isBedManagementModuleInstalled = useFeatureFlag('bedmanagement-module');
   const locationFromUrl = allLocations.find((l) => l.uuid === locationUuidFromUrl);
-
   const invalidLocation = locationUuidFromUrl && !locationFromUrl;
   const location = (locationFromUrl ?? sessionLocation) as any as Location;
 
+  if (!isBedManagementModuleInstalled) return <></>;
```

Comment:
Maybe just add a TODO comment here to say "Display patients with admitted status (based on their observations) that have no beds assigned". (We don't have the API to retrieve this info yet)